code/style.css

```diff
@@ -1,5 +1,465 @@
-/* After you've linked the CSS file in the HTML,
-this should turn the background blue ;) */
+@import url('https://fonts.googleapis.com/css2?family=Cinzel:wght@400..900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
```

Comment:
Since you are using Cinzel as font in most places consider adding it as a font-family on the body instead of declaring it on multiple elements. Then you can declare the other font, Oswald, on the specific places you want it. On all headings for instance.